salt/daemons/flo/jobber.py

```diff
@@ -307,13 +307,14 @@ def proc_run(self, msg):
                         'non-empty list expected'.format(executors)
                     )
                 if self.opts.get('sudo_user', '') and executors[-1] != 'sudo':
-                    executors[-1] = 'sudo.get'  # replace
+                    executors[-1] = 'sudo'  # replace
                 log.trace("Executors list %s", executors)
 
                 for name in executors:
-                    if name not in self.module_executors.value:
+                    fname = '{0}.execute'.format(name)
+                    if fname not in self.module_executors.value:
                         raise SaltInvocationError("Executor '{0}' is not available".format(name))
-                    return_data = self.module_executors.value[name].execute(self.opts, data, func, args, kwargs)
+                    return_data = self.module_executors.value[fname](self.opts, data, func, args, kwargs)
```

Comment:
I'm not familiar with executors, is this a bugfix or is `.execute()` equivalent to `.__call__()`?

Reply:
It's a bugfix.